### ISSUE TITLE
Not getting intent

### ISSUE THREAD

**Issue by petec029** [BeepBoopHQ/witbot#3]
I am trying to use witbot for the first time and can not get witbot to recognize the intent.  I am wondering if this is because the message structure has changed

API

Version change

On 2016, May 11th, the /message API was updated to reflect the new Bot Engine model: intent are now entities. We updated the SDK to the latest version: 20160516. You can target a specific version by setting the env variable WIT_API_VERSION.

{
  "msg_id" : "e86468e5-b9e8-4645-95ce-b41a66fda88d",
  "_text" : "hello",
  "entities" : {
    "intent" : [ {
      "confidence" : 0.9753469589149633,
      "value" : "greetings"
    } ]
  }
}

**Comment by mbrevoort** [comment]
Yes, unfortunately wit.ai has changed considerably with they launched Bot
Builder.

**Comment by petec029** [comment]
Are you working on a more comprehensive rewrite that handles their new bot creation tools?
In the meantime if it is simple do you have time to fix it to just parse the new json structure to get the intent?

**Comment by localjo** [comment]
Are there any plans to update witbot to use the new Wit.ai API? Looks like my best course of action right now is to find a different way to interface with Wit.ai.
